packages/clients/textLocator/src/utils/textLocatorBackend/findLiterature/searchLiterature.ts

```diff
@@ -0,0 +1,66 @@
+import { CoreGetters, CoreState, PolarStore } from '@polar/lib-custom-types'
+import { FeatureCollection, Point } from 'geojson'
+import { TitleLocationFrequency } from '../../../types'
+import urlSuffix from '../urlSuffix'
+
+const mapResponseToFeatureCollection = (
+  titleLocationFrequency: TitleLocationFrequency
+): FeatureCollection<Point, Record<string, number>> => ({
+  type: 'FeatureCollection',
+  features: Object.entries(titleLocationFrequency).map(
+    ([literatureId, { title, location_frequency: locationFrequency }]) => ({
+      type: 'Feature',
+      // fake geom to fit APIs; ignored by custom selectLiterature
+      geometry: { type: 'Point', coordinates: [0, 0] },
+      properties: locationFrequency,
+      epsg: null,
```

Comment:
I don't remember, but we have the same pattern in DISH, so I assume it's a check for `null` somewhere that goes wrong for `undefined`. Since this is the tested variant, I'd like to simply keep it this way. Should you see a greater need to remove this maybe unnecessary addition or discover why there's a need for it, feel free to remove it and test in depth.

Reply:
Removed it as it still works after removal c5f42d92530e49388a5c1d4c051eef4a383138a5 🎩